app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
```

Comment:
OK.  We might add a ticket at some point to parameterize the name of the site.  It's hardcoded as *DPLA* Primary Source Sets